src/components/JournalEntry/EditJournalEntry.tsx

```diff
@@ -124,15 +126,71 @@ const extensions = [
   StarterKit,
 ];
 
-export const EditJournalEntry = () => {
+export const EditJournalEntry = ({
+  onSave,
+}: {
+  onSave: (boxContent: JSONContent) => void;
+}) => {
+  const editorRef = useRef<Editor | null>(null);
+
+  const [wizard, setWizard] = useState<
+    | {
+        type: "initial";
+      }
+    | {
+        type: "loading";
+      }
+    | {
+        type: "error";
+        message: string;
+      }
+  >({ type: "initial" });
+
+  const sendEntry = () => {
+    setWizard({ type: "loading" });
+    const saveAndClear = () => {
+      if (editorRef.current !== null && editorRef.current.isEmpty === false) {
+        onSave(editorRef.current.getJSON());
+        editorRef.current.commands.clearContent();
+      }
+      setWizard({ type: "initial" });
+    };
+
+    const entryPromise = new Promise<void>((resolve) => {
+      setTimeout(() => {
+        resolve();
+      }, 1000);
+    });
+
+    entryPromise.then(() => {
+      saveAndClear();
+    });
+  };
+
   return (
-    <div className="m-3 rounded-md border-black border-solid border-2 sm:w-8/12">
+    <div className="m-3 rounded-md border-black border-solid border-2 sm:w-8/12 w-5/6">
```

Comment:
As a general rule, components should not have their own `width`. Instead, the parent should set their width.

This is helpful, because we can use the same component in multiple place. So here, remove the `w-5/6` and send it from the parent via a `className` component.